docker-compose.yml

```diff
@@ -0,0 +1,103 @@
+version: '3'
+services:
+  node-0:
+    image: cennznet/cennznet:1.1.1
+    volumes:
+    - ./data/node-0:/mnt/data
+    command:
+      - --chain=/cennznet/genesis/nikau.raw.json
+      - --validator
+      - --alice
+      - --base-path=/mnt/data
+      - --name=alice-${HOSTNAME:-localhost}-0
+      - --unsafe-ws-external
+      - --unsafe-rpc-external
+      - --rpc-cors=all
```

Comment:
just a doubt - why we don't need **node-key** here, we have it for all other service?

Reply:
It sets the node's network level public key. It's mostly important for hardcoding bootnodes here